modules/openy_features/openy_prgf/modules/openy_prgf_featured_news/openy_prgf_featured_news.info.yml

```diff
@@ -0,0 +1,11 @@
+name: 'OpenY paragraph featured news posts (Experimental)'
```

Comment:
Remove `(Experimental)` from module name here and in other modules.
Instead add package - `package: OpenY (Experimental)` to all modules